src/software/ai/hl/stp/play/halt_play_test.cpp

```diff
@@ -33,10 +34,7 @@ TEST_F(HaltPlayTest, test_halt_play)
         // TODO: Implement proper validation
         // https://github.com/UBC-Thunderbots/Software/issues/1971
```

Comment:
please remove this todo and initialize the robots with some velocities

Reply:
Created a new function to initialize velocities and used that in the test